polytope_server/common/staging/s3_boto3_staging.py

```diff
@@ -76,17 +74,9 @@ def __init__(self, config):
         for name in ["boto", "urllib3", "s3transfer", "boto3", "botocore", "nose"]:
             logging.getLogger(name).setLevel(logging.WARNING)
 
-        prefix = "https" if self.use_ssl else "http"
-
-        if config.get("random_host", False):
-            self.host = config.get("random_host", {}).get("host", self.host)
-            index = random.randint(0, config.get("random_host", {}).get("max", 1) - 1)
-            # replace %%ID%% in the host with the index
-            self.host = self.host.replace("%%ID%%", str(index))
-            self.url = self.url + "/" + str(index)
-            logging.info(f"Using random host: {self.host}")
+        self.prefix = "https" if self.use_ssl else "http"
 
-        self._internal_url = f"{prefix}://{self.host}:{self.port}"
+        self._internal_url = f"http://{self.host}:{self.port}"
```

Comment:
Is this prefix intentionally fixed now?

Reply:
This is for internal communication, and I do not believe we need to use https there at all. The use_ssl setting in the configuration should only affect download urls, not internal communication.